src/test/kotlin/fr/social/gouv/agora/usecase/qagPreview/GetQagPreviewListUseCaseTest.kt

```diff
@@ -112,28 +112,32 @@ internal class GetQagPreviewListUseCaseTest {
     }
 
     @Test
-    fun `getQagPreviewList - when getQagListUseCase returns qags - should return supportedPreviewList with supportedQag only sorted by supportDate desc`() {
+    fun `getQagPreviewList - when getQagListUseCase returns qags - should return supportedPreviewList with supportedQag only with posted qag sorted by postDate`() {
```

Comment:
`supportedPreviewList with supportedQag only with posted qag sorted by postDate`
Je crois qu'on s'est mal compris, j'aurai plutôt mis:
`should return posted qags sorted by postDate followed by supportedQag sorted by supportDate desc`

Reply:
done